install_scripts/helpers.py

```diff
@@ -131,6 +136,13 @@ def get_default_docker_version():
         default=_default_docker_version)
 
 
+def get_default_kubernetes_version():
+    return param.lookup(
+        'PINNED_KUBERNETES_VERSION',
+        '/install_scripts/pinned_kubernetes_version',
```

Comment:
This param doesn't exist, right?

Reply:
no and hopefully it doesnt matter. it makes it easier to run in dev